categories.xml

```diff
@@ -37,58 +37,75 @@
     </category>
     <category name="Deprecated 1.8" slug="deprecated-1.8">
-      <desc><![CDATA[All the aspects of the API that were deprecated in the corresponding version of jQuery.
+      <desc><![CDATA[
+        <div id="version-support-warning" class="warning"><i class="icon-info-sign"></i> <span>This version is no longer supported. Read more about our <a href="https://jquery.com/support/">Version Support</a>.</span></div>
+        <p>All the aspects of the API that were deprecated in the corresponding version of jQuery.</p>
         <p>For more information, see the Release Notes/Changelog at <a href="https://blog.jquery.com/2012/08/09/jquery-1-8-released/">https://blog.jquery.com/2012/08/09/jquery-1-8-released/</a></p>
       ]]></desc>
     </category>
     <category name="Deprecated 1.9" slug="deprecated-1.9">
-      <desc><![CDATA[All the aspects of the API that were deprecated in the corresponding version of jQuery.
+      <desc><![CDATA[
+        <div id="version-support-warning" class="warning"><i class="icon-info-sign"></i> <span>This version is no longer supported. Read more about our <a href="https://jquery.com/support/">Version Support</a>.</span></div>
+        <p>All the aspects of the API that were deprecated in the corresponding version of jQuery.</p>
         <p>For more information, see the Release Notes/Changelog at <a href="https://blog.jquery.com/2013/01/15/jquery-1-9-final-jquery-2-0-beta-migrate-final-released/">https://blog.jquery.com/2013/01/15/jquery-1-9-final-jquery-2-0-beta-migrate-final-released/</a></p>
       ]]></desc>
     </category>
-    <category name="Deprecated 1.10" slug="deprecated-1.10-and-2.0">
-      <desc><![CDATA[All the aspects of the API that were deprecated in the corresponding version of jQuery.
+    <category name="Deprecated 1.10 &amp; 2.0" slug="deprecated-1.10-and-2.0">
+      <desc><![CDATA[
+        <div id="version-support-warning" class="warning"><i class="icon-info-sign"></i> <span>These versions are no longer supported. Read more about our <a href="https://jquery.com/support/">Version Support</a>.</span></div>
+        <p>All the aspects of the API that were deprecated in the corresponding version of jQuery.</p>
         <p>For more information, see the Release Notes/Changelog at <a href="https://blog.jquery.com/2013/05/24/jquery-1-10-0-and-2-0-1-released/">https://blog.jquery.com/2013/05/24/jquery-1-10-0-and-2-0-1-released/</a></p>
       ]]></desc>
     </category>
     <category name="Deprecated 3.0" slug="deprecated-3.0">
-      <desc><![CDATA[All the aspects of the API that were deprecated in the corresponding version of jQuery.
+      <desc><![CDATA[
         <p>For more information, see the Release Notes/Changelog at <a href="https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/">https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/</a></p>
+        <p>All the aspects of the API that were deprecated in the corresponding version of jQuery.</p>
```

Comment:
The above "For more information" sounds a bit weird when it's the first sentence.

This applies to many places.

Reply:
Oops, that was unintentional.